src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -139,6 +139,16 @@
     short-summary: Manage file storage directory metadata.
 """
 
+helps['storage directory list'] = """
+    type: command
+    short-summary: List directories in the specified share.
+    parameters:
+        - name: --num-results
+          type: integer
+          short-summary: Specifies the maximum number of directories to return. The default value is
+                         5000. The value cannot be less or equal to zero.
```

Comment:
If this is used as the input to the SDK call to get files and directories, then this is not accurate help text because any files (which will be filtered out) would count against the 5000 count quota.